UnofficialSteamAuthenticator/MainPage.xaml.cs

```diff
@@ -128,24 +123,31 @@ internal void HandleUri(Uri uri)
 
         private void steamGuardUpdate_Tick(object sender, object e)
         {
-            if (storyboard != null) storyboard.Stop();
-            if (SteamGuardGrid.Visibility != Visibility.Visible) return;
+            if (_storyboard != null)
+            {
+                _storyboard.Stop();
+            }
+
+            if (SteamGuardGrid.Visibility != Visibility.Visible)
+            {
+                return;
+            }
 
             TimeAligner.GetSteamTime(async time =>
             {
-                ulong currentChunk = (ulong)time / 30L;
+                //ulong currentChunk = (ulong)time / 30L;
```

Comment:
You're right. Thought you maybe had something in mind with it, so I merely commented it.